docs/02.API-REFERENCE.md

```diff
@@ -4736,6 +4736,77 @@ main (void)
 - [jerry_init](#jerry_init)
 - [jerry_cleanup](#jerry_cleanup)
 - [jerry_exec_snapshot](#jerry_exec_snapshot)
+- [jerry_parse_and_save_static_snapshot](#jerry_parse_and_save_static_snapshot)
+
+
+## jerry_parse_and_save_static_snapshot
+
+**Summary**
+
+Generate static snapshot from the specified source code.
+
+Unlike normal snaphots static snaphots are fully executed from ROM. Not
+even their header is loaded into the RAM. However they can only depend
+on magic strings and 28 bit integer numbers. Regular expression literals
```

Comment:
I think we should emphasise that the magic strings during the snapshot generation must be the same as the magic strings during execution.